From: Pierre Habouzit <madcoder@debian.org>
Date: Tue, 20 Nov 2007 11:22:05 +0100
Subject: [PATCH] pkg-config.pc.in: move @LIBS@ in the Libs.private section

Also avoid to link binaries against libz, libm and friends.

Signed-off-by: Pierre Habouzit <madcoder@debian.org>
---
 kyotocabinet.pc.in |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/kyotocabinet.pc.in b/kyotocabinet.pc.in
index 06e2276..a445920 100644
--- a/kyotocabinet.pc.in
+++ b/kyotocabinet.pc.in
@@ -10,5 +10,6 @@ datadir=@datadir@
 Name: Kyoto Cabinet
 Description: a straightforward implementation of DBM
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lkyotocabinet @LIBS@
+Libs: -L${libdir} -lkyotocabinet
+Libs.private: @LIBS@
 Cflags: -I${includedir}
-- 
